client/components/VideoChat.jsx

```diff
@@ -15,7 +15,7 @@ class VideoChat extends React.Component {
 
   componentDidMount() {
     const constraints = {
-      audio: true,
+      audio: false, // ZS set this to false so that annyang can access the mic
```

Comment:
yea we thought that would be cool for a future iteration, just logged it on github. it might be possible but to minimize variables we disabled it for now until we prove annyang works on prod